assignments/callbacks.js

```diff
@@ -25,27 +25,63 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 */
 
 
-function getLength(arr, cb) {
+function gettingLength(arr, cb){
+  return cb(arr);
   // getLength passes the length of the array into the callback.
 }
 
+function itemsLength(items){
+  return items.length;
+}
+console.log(gettingLength(items, itemsLength));
+
 function last(arr, cb) {
+  return cb(arr);
   // last passes the last item of the array into the callback.
 }
 
+function lastArr(items){
+  return items.length - 1;
+}
```

Comment:
you'll want this to return the contents in the last index, correct? If so, how is that done? Right now, you are returning the last index of the array.